packages/docusaurus-theme-search/src/theme/SearchBar/index.module.css

```diff
@@ -0,0 +1,703 @@
+  border-bottom-color: var(--markprompt-border);
+}
+.MarkpromptAnswer thead th {
+  color: var(--markprompt-foreground);
+  font-weight: 600;
+  vertical-align: bottom;
+  padding-right: 1em;
+  padding-bottom: 0.6666667em;
+  padding-left: 1em;
+}
+.MarkpromptAnswer tbody tr {
+  border-bottom-width: 1px;
+  border-bottom-color: var(--markprompt-border);
+}
+.MarkpromptAnswer tbody tr:last-child {
+  border-bottom-width: 0;
+}
+.MarkpromptAnswer tbody td {
+  vertical-align: baseline;
+}
+.MarkpromptAnswer tfoot {
+  border-top-width: 1px;
+  border-top-color: var(--markprompt-border);
+}
+.MarkpromptAnswer tfoot td {
+  vertical-align: top;
+}
+.MarkpromptAnswer video {
+  margin-top: 1.7142857em;
+  margin-bottom: 1.7142857em;
+}
+.MarkpromptAnswer figure {
+  margin-top: 1.7142857em;
+  margin-bottom: 1.7142857em;
+}
+.MarkpromptAnswer li {
+  margin-top: 0.2857143em;
+  margin-bottom: 0.2857143em;
+}
+.MarkpromptAnswer ol > li {
+  padding-left: 0.4285714em;
+}
+.MarkpromptAnswer ul > li {
+  padding-left: 0.4285714em;
+}
+.MarkpromptAnswer > ul > li p {
+  margin-top: 0.5714286em;
+  margin-bottom: 0.5714286em;
+}
+.MarkpromptAnswer > ul > li > *:first-child {
+  margin-top: 1.1428571em;
+}
+.MarkpromptAnswer > ul > li > *:last-child {
+  margin-bottom: 1.1428571em;
+}
+.MarkpromptAnswer > ol > li > *:first-child {
+  margin-top: 1.1428571em;
+}
+.MarkpromptAnswer > ol > li > *:last-child {
+  margin-bottom: 1.1428571em;
+}
+.MarkpromptAnswer ul ul,
+.MarkpromptAnswer ul ol,
+.MarkpromptAnswer ol ul,
+.MarkpromptAnswer ol ol {
+  margin-top: 0.5714286em;
+  margin-bottom: 0.5714286em;
+}
+.MarkpromptAnswer hr + * {
+  margin-top: 0;
+}
+.MarkpromptAnswer h2 + * {
+  margin-top: 0;
+}
+.MarkpromptAnswer h3 + * {
+  margin-top: 0;
+}
+.MarkpromptAnswer h4 + * {
+  margin-top: 0;
+}
+.MarkpromptAnswer thead th:first-child {
+  padding-left: 0;
+}
+.MarkpromptAnswer thead th:last-child {
+  padding-right: 0;
+}
+.MarkpromptAnswer tbody td,
+.MarkpromptAnswer tfoot td {
+  padding-top: 0.6666667em;
+  padding-right: 1em;
+  padding-bottom: 0.6666667em;
+  padding-left: 1em;
+}
+.MarkpromptAnswer tbody td:first-child,
+.MarkpromptAnswer tfoot td:first-child {
+  padding-left: 0;
+}
+.MarkpromptAnswer tbody td:last-child,
+.MarkpromptAnswer tfoot td:last-child {
+  padding-right: 0;
+}
+
+.caret {
+  display: none;
+  height: 1rem;
+  width: 0.5rem;
+  margin-left: 0.2rem;
+  transform: translate(2px, 2px);
+  border-radius: 1px;
+  background-color: var(--markprompt-primary);
+  box-shadow: 0 0 3px 0 var(--markprompt-primary);
+  animation-name: blink;
+  animation-duration: 1000ms;
+  animation-fill-mode: both;
+  animation-iteration-count: infinite;
+  transition-timing-function: cubic-bezier(0.14, 0, 0.16, 1);
+}
+
+[data-loading-state='preload'] .caret {
+  display: inline-block;
+}
+
+[data-loading-state]:not([data-loading-state='done']) .caret {
+  display: none;
+}
+
+[data-loading-state='preload'] .progress {
```

Comment:
I had missed this change but to me `preload` suggests something else than the `indeterminate` that was there before. Preloading is fetching data before we needed. If `indeterminate` isn't correct enough, I would suggest `beforeload` maybe?